commercetools-sdk-base/src/main/java/io/sphere/sdk/search/model/FilteredFacetExpressionImpl.java

```diff
@@ -25,4 +25,9 @@ public boolean equals(Object o) {
     public String alias() {
         return super.alias();
     }
+
+    @Override
```

Comment:
if it is for the IDE highlighting it should not be in the impl class but on the class you use